src/lib/pay-request/utils/error.ts

```diff
@@ -8,3 +9,16 @@ export function handleEntityNotFound(entityName: string, entityId: string) {
     throw error
   }
 }
+
+export function handleChargeNotFoundForParityCheck(entityName: string, entityId: string, parityCheck = false) {
+  return (error: RESTClientError) => {
+    if (error.data.response && error.data.response.status === 404) {
+      if (parityCheck) {
+        const chargeNotFound: Charge = { charge_id: 'Not found', amount: undefined, total_amount: undefined, fee: undefined, net_amount: undefined, description: undefined, reference: undefined, payment_provider: undefined }
```

Comment:
is it necessary to return a Charge object here if the only data being passed back to the call site is `'Not found'`?

Reply:
Fixed